src/main/java/org/elasticsearch/index/analysis/SkroutzGreekStemmer.java

```diff
@@ -1,19 +1,12 @@
 package org.elasticsearch.index.analysis;
 
-import java.io.IOException;
-import java.io.InputStream;
-
-import java.util.Map;
 import java.util.Arrays;
-import java.util.List;
 
 import org.apache.lucene.analysis.el.GreekLowerCaseFilter;
-import org.apache.lucene.analysis.CharArrayMap;
 import org.apache.lucene.analysis.CharArraySet;
 
 import org.apache.logging.log4j.Logger;
 import org.elasticsearch.common.logging.ESLoggerFactory;
-import org.yaml.snakeyaml.Yaml;
```

Comment:
[?] how is this not a dependency found in  `pom.xml` ? :-|

Reply:
Good question. Because it comes from `org.elasticsearch`:
```
[INFO] gr.skroutz:elasticsearch-skroutz-greekstemmer:jar:5.4.2.6                                         
[INFO] +- org.elasticsearch:elasticsearch:jar:5.4.2:compile                            
[INFO] |  +- org.apache.lucene:lucene-core:jar:6.5.1:compile                                             
[INFO] |  +- org.apache.lucene:lucene-analyzers-common:jar:6.5.1:compile                                 
[INFO] |  +- org.apache.lucene:lucene-backward-codecs:jar:6.5.1:compile                                  
[INFO] |  +- org.apache.lucene:lucene-grouping:jar:6.5.1:compile                                         
[INFO] |  +- org.apache.lucene:lucene-highlighter:jar:6.5.1:compile                                      
[INFO] |  +- org.apache.lucene:lucene-join:jar:6.5.1:compile                                             
[INFO] |  +- org.apache.lucene:lucene-memory:jar:6.5.1:compile                                           
[INFO] |  +- org.apache.lucene:lucene-misc:jar:6.5.1:compile                                             
[INFO] |  +- org.apache.lucene:lucene-queries:jar:6.5.1:compile                                          
[INFO] |  +- org.apache.lucene:lucene-queryparser:jar:6.5.1:compile                                      
[INFO] |  +- org.apache.lucene:lucene-sandbox:jar:6.5.1:compile
[INFO] |  +- org.apache.lucene:lucene-spatial:jar:6.5.1:compile                                          
[INFO] |  +- org.apache.lucene:lucene-spatial-extras:jar:6.5.1:compile
[INFO] |  +- org.apache.lucene:lucene-spatial3d:jar:6.5.1:compile                                        
[INFO] |  +- org.apache.lucene:lucene-suggest:jar:6.5.1:compile
[INFO] |  +- org.elasticsearch:securesm:jar:1.1:compile                                                  
[INFO] |  +- net.sf.jopt-simple:jopt-simple:jar:5.0.2:compile                                            
[INFO] |  +- com.carrotsearch:hppc:jar:0.7.1:compile                                                     
[INFO] |  +- joda-time:joda-time:jar:2.9.5:compile                                                       
[INFO] |  +- org.yaml:snakeyaml:jar:1.15:compile
...
```
As the package had been used in `SkroutzGreekStemmer` directly, the dependency should be explicitly declared. In any case, now there is no such dependency at all, and `pom.xml` remains as is... :)